screens/RegisterName.js

```diff
@@ -46,6 +46,11 @@ export default function RegisterName({ name }) {
   );
 }
 
+RegisterName.propTypes = {
+  name: PropTypes.string,
+  navigatiom: PropTypes.func,
```

Comment:
I don't think navigation is a prop. Also it's spelled wrong but probably irrelevant.